styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+  background-color: #00cdcd;
+}
+
+.nav-items {
+  list-style: none;
+  padding: 0;
+  float: right;
+}
+
+.nav-item {
+  display: inline-block;
+}
+
+.nav-link {
+  padding: 20px;
+  color: #ffffff;
+  text-decoration: none;
+  display: block;
+  border-bottom: 2px solid transparent ;
+}
+
+.nav-link:hover {
+  border-bottom: 2px solid #ffffff;
+  }
+.dropdown-toggle{
+  position: relative;
+}
+.dropdown-content{
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  position: absolute;
+  display: none;
+  text-decoration: none;
```

Comment:
e có thể viết a {
  text-decoration: none;
} là nó sẽ apply vào toàn bộ thẻ a của trang, đỡ phải viết lại vào từng class.
tương tự như thế vs cái ul và list-style ở dưới